README.md

```diff
@@ -383,10 +383,37 @@ circleci config validate .circleci/config.yml
 
 ## Developing updates for this gem
 
+There are 2 ways to validate and test changes to this orb.
+
+### Publishing the orb as a dev version
+
 To push the current code into the `dev` version online:
 
 `circleci orb publish src/standard.yml tablexi/standard@dev:first`
 
+You can then test/validate by using the orb in your app's `config.yml`:
+
+```
+orbs:
+  tablexi: tablexi/standard@dev:first
+```
+
+### Using config packing to make it an inline orb
+- Copy your app's `.circleci/config.yml` to `src/config.yml`
+- Add `orbs`/`tablexi` keys to the top level of `src/standard.yml`:
+  ```
+  orbs:
+    tablexi:
+      description: "A set of standard steps which we use for projects at Table XI"
```

Comment:
This is not clear...do you mean to say that I should indent the whole existing contents of `standard.yml` to be 4 spaces indented, so that the contents are beneath the `orbs/tablexi` headers?

Reply:
Yeah, that's right.  Not the most user friendly but it works.  

There is another option, which would be creating a separate local testing directory:
```
local-testing/orbs/tablexi.yml
local-testing/config.yml
```

To test locally:
- Copy your app's `.circleci/config.yml` to `local-testing/config.yml`
- `cp standard.yml local-testing/orbs/tablexi.yml`
- To validate, run `circleci config pack local-testing | circleci config validate -`